python/ql/src/semmle/python/ApiGraphs.qll

```diff
@@ -6,7 +6,7 @@
  * directed and labeled; they specify how the components represented by nodes relate to each other.
  */
 
-import python
+private import python
```

Comment:
This one puzzles me a bit. This would only affect cases where we do `import semmle.python.ApiGraphs` but not `import python`, no?

Reply:
Indeed. If anyone has written a query _without_ `import python`, things will go on fire :fire:

Since things can go wrong with stage re-evaluation if you don't start your query with `import python`, I'm actually quite ok with accepting those risks :+1:

Reply:
Oh, sure. In fact, I think we should be using `private import` a lot more, as it helps to not pollute the namespace. (This would, for instance, have allowed us to more cleanly add `DataFlow::CallNode` without clashing witn the corresponding control flow node.)

I just don't see how this would affect auto-complete in any meaningful way. :slightly_smiling_face:

Reply:
Hmm, it's seems like you're right.

Without this PR, I get bad auto-complete suggestions for `DataFlow::`

![image](https://user-images.githubusercontent.com/1054041/112135609-40b41680-8bce-11eb-8b84-298904a1f1f8.png)

But the ones for `API::` actually look good :thinking: 

![image](https://user-images.githubusercontent.com/1054041/112135650-4b6eab80-8bce-11eb-8cfd-313dfc2d4e95.png)

So while my argument for _this_ part of the PR might be a bit wrong, I still think it's good to do :+1: (for the stage re-evaluation reasons)